src/sample/Controller.java

```diff
@@ -35,7 +48,7 @@ public static void setPrevStage(Stage stage)
     }
 
     public void handleButtonAction()
-    {
+    { System.out.println("it works");
```

Comment:
Remove all print statements.  They should be for debugging only and not end up in main.